DemoApp/Sources/Components/MicrophoneChecking/MicrophoneChecking.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  MicrophoneChecking.swift
+//  StreamVideoCallCore
+//
+//  Created by Ilias Pavlidakis on 2/6/23.
+//
+
+import Foundation
+import Combine
+
+protocol MicrophoneChecking {
```

Comment:
Why do we need this?

Reply:
It's not needed any more. It was used when we had the slim version of the SwiftUI SDK